cosmos/models/Workflow.py

```diff
@@ -552,7 +552,7 @@ def _run(workflow, session, task_queue):
 
         # only commit Task changes after processing a batch of finished ones
         session.commit()
-        time.sleep(.3)
+        time.sleep(5)
```

Comment:
Actually this is the more important one -- the previous timeout change only modifies Cosmos's behavior in the error case, this affects it in the common case.

However, I see your point. Just because UGE can't handle a fast polling interval doesn't mean every drm should suffer.

Seems to me this could be handled reasonably cleanly if each drm had a POLL_INTERVAL field, where `local`'s was 0.3, `ge`'s and `lsf`'s were 5 or 10. Since the job manager keeps a list of outstanding tasks, you'd only need to reduce your polling interval when you were waiting on a cluster job. Then we'd replace this line with something like this --

`time.sleep(workflow.jobmanager.poll_interval())`